src/admin/__tests__/deleteTopics.spec.js

```diff
@@ -102,7 +102,9 @@ describe('Admin', () => {
       )
 
       expect(broker.deleteTopics).toHaveBeenCalledTimes(1)
-      expect(loggerInstance.error).toHaveBeenCalledWith(
+      expect(
```

Comment:
Could you help me with this, I had a lot of errors with code linting, then I run:
npm run lint -- --fix
Now testing passes when I run:
sudo yarn test
So I don't know why it's saying that it isn't ok on your side...

Reply:
I pushed a commit to your branch that fixes the linting. `yarn lint` shows you the errors and `yarn lint --fix` fixed them.

> Now testing passes when I run:
> sudo yarn test

No need to run yarn with sudo

Reply:
Hm, I pulled your changes, now I run `yarn lint` and get a bunch of errors:
`/home/dfilkovi/work/env_leapbit/kafkajs/src/consumer/offsetManager/__tests__/initializeConsumerOffsets.spec.js
  54:22  error  Replace `{·partition:·0,·offset:·'1'·},·{·partition:·1,·offset:·'2'·}` with `⏎··········{·partition:·0,·offset:·'1'·},⏎··········{·partition:·1,·offset:·'2'·},⏎········`  prettier/prettier

/home/dfilkovi/work/env_leapbit/kafkajs/src/admin/__tests__/deleteTopics.spec.js
  105:14  error  Replace `loggerInstance.error` with `⏎········loggerInstance.error⏎······`  prettier/prettier

✖ 14 problems (14 errors, 0 warnings)
  14 errors and 0 warnings potentially fixable with the `--fix` option.

`

I think if I run fix, then your side will say that linting is not ok, I would really like to fix this as I already made another method to delete consumers, and I am working on a desktop GUI tool for kafka that is using your library which will soon see the light of the day...

Reply:
Yeah, there's clearly some environment issue going on. Did you install dependencies using `yarn`? If you installed them using `npm`, it could be that you got different versions of some packages, since there's no lockfile for it. That could explain why you are getting different results than in CI.

Reply:
Yep it was my install via npm not yarn causing the issue